common/subdevice-model.cpp

```diff
@@ -1752,7 +1752,10 @@ namespace rs2
         bool supported_fw = s->supports(RS2_CAMERA_INFO_FIRMWARE_VERSION) ?
             is_upgradeable("05.11.12.0", fw_version) : false;
 
-        return s->is<rs2::depth_sensor>() && is_d400 && supported_fw;
+        bool is_d500 = s->supports(RS2_CAMERA_INFO_PRODUCT_LINE) ?
+            std::string(s->get_info(RS2_CAMERA_INFO_PRODUCT_LINE)) == "D500" : false;
+
+        return (s->is<rs2::depth_sensor>() && is_d400 && supported_fw) || (s->is<rs2::depth_sensor>() && is_d500);
```

Comment:
Maybe better to wrap d400 related in another brackets?
or split to separate local variables and check them?

```
auto d400_calib_supported = ...
auto d500_calib_supported = ...
return d400_calib_supported  || d500_calib_supported ;
```

What do you think?